selfapi.js

```diff
@@ -555,13 +593,21 @@ function getHandlerExporter (app) {
   // `app` is an express-like server app.
   return function (method, path, parameters) {
     // Support restify.
-    if (method === 'delete' && ('del' in app)) {
-      method = 'del';
+    if (method === 'del' && ('delete' in app)) {
+      method = 'delete';
     }
     app[method](path, parameters.handler);
   };
 }
 
+// Stringify object, pass down strings
+function jsonStringify(strOrObject) {
+  if (strOrObject instanceof Object) {
+    return JSON.stringify(strOrObject, options.jsonStringifyReplacer, options.jsonStringifySpaces);
```

Comment:
Bonus mega-nit: Doesn't this function exceed 80 chars? I agree that this limit is stupid, but somehow I keep trying to respect it. Maybe it's because of people like me that we don't move into the future faster.